### DISCUSSION TITLE
Problem with message signing in Flutter

### DISCUSSION THREAD

**Discussion by tanmayjangde** [WalletConnect/walletconnect-monorepo#3009] [Developer Support]
Can I get correct flow for connecting metamask wallet and signing message using metamask and walletconnect_flutter_v2 with code?

**Comment by crypblizz8** [comment]
Hey there, have you checked out our examples here: https://github.com/WalletConnect/WalletConnectFlutterV2/tree/master/example ?

**Reply by tanmayjangde** [reply]
Yes, I did. But it is still not clear how can we sign a message with metamask or any other wallet. Can you give me exact code which will be used for signing a message?
